c_library/gateway_c.cpp

```diff
@@ -109,3 +109,41 @@ const char* MuxyGateway_GetProductionURL(MuxyGateway Gateway)
 	gateway::SDK* SDK = static_cast<gateway::SDK*>(Gateway.SDK);
     return SDK->GetProductionURL().c_str();
 }
+
+void MuxyGateway_RunPoll(MuxyGateway Gateway, GW_PollConfiguration cfg)
```

Comment:
Ticket says this function should be called `StartPoll` is there a reason you think `RunPoll` works better? https://muxy.atlassian.net/browse/PONY-139

Reply:
I think I was referencing the existing RunPoll functionality, but the docs do say it should be called Start/Stop poll.